### DISCUSSION TITLE
Meteorb Script

### DISCUSSION THREAD

**Discussion by sifter456** [isid0re/SoloLeveling#78] [Q&A] [answered]
Posted in issues will post here also, anyone know why everytime im trying to make the SoloLevel use the meteorb script no matter what i do i took the dashes out for meteorb and everything and it still keeps using blizzballer everytime after the first spec and keeps loading blizzballer script. getting frustrated would really just like some help as i hate using a blizzard sorc

**Comment by isid0re** [comment, marked as answer]
My script works differently than other autoplays because it operates with 2 respecs. 

Sorceress progression is charged bolt to blizzballer then to the  final build choice. In your case meteorb.

**Reply by sifter456** [reply]
I also found out how to make it use meteorb as the middle build in the config file if you scroll down a ways theres a section to change the middle tier build which is set to blizzballer like you suggested. i changed ti to meteorb , but i notice while i attacking it mainly spams meteor and fireball and not really FO....ill have too see what i can do to change that up a little too so that way i dont rip when i get to hell lol

**Reply by isid0re** [reply]
Sounds like you want to have it be an orbital instead of a meteorb. (Where primary timed is orb vs meteor) Is so, you will need to change the entire build around then.

1. You will need to change the skill progression to have it optimized by level and damage. 

2. Adjust the autobuild template for the untimed skill for frozen orb instead of meteor.

**Reply by sifter456** [reply]
Yes i got it changed to select orb max and the Syns first then work on meteor, im still working on getting the untimed skill changed. i just started botting again after a few year hiatus but ill get it figured out, i also and am trying to find a pickit to use with these scripts that it will keep stuff, because sometimes it will throw good things out  IE. Trang gloves, and an RFrost

**Reply by isid0re** [reply]
Regarding my logic for using blizzballer for leveling check this link.

https://www.diabloii.net/forums/threads/meteorb-vs-orbitaller.814547/

As for pickit items. I use a dynamic calculator called tierscore (found in globals.js) it evals the weights based on score and needed resists with a focus on survivability. So if it drops items then the score was deemed needed vs another item. 

For example, let's say it's running max resists without using its armors stats. The bot wont rate smoke highly since the resists are already maxed out. So stealth would actually worth more.

But if you want to keep it stil then make a seperate mf pickit.

**Reply by sifter456** [reply]
Yeah but thing i dont get, like i just noticed. my pally is in hell and with smoke on he has almost all max resist, but when it takes smoke off they are all pretty low like 40's ish. but it will take the smoke off and then throw it on floor and buy a armor with 100 life and no res lol im trying to figure that out.

**Reply by sifter456** [reply]
and without the resist it needs its causing it too chicken more frequently cause chickens are set high cause im on hardcore. i know these level bots are mainly designed for softcore builds but ive been able to succeed thus far with a lot of trial and error, ive had five rips so far but have adjusted Chickens and lvl progression to move to new difficulties accordingly. i think i have it down now as all 4 chars are almost lvl 75+ with the druid being the highest and living the longest at lvl 87. and this is starting with absolutely nothing ive had no gear or anything its all been done by them. i actually just made my first hoto cuz the pally found 2 guls and made vex and ive found a few puls and etc. so its been succesful this far just some of the things it does mind boggles me lol

**Reply by sifter456** [reply]
Ohhh i also see you just updated the scripts 4 days ago with some HC Settings, what all did you include/ change in them as i cant see exactly what not 100% sure as of what just to make sure it wasnt things i tweaked myself.

**Reply by sifter456** [reply]
Ya i downloaded the new files and installed. only problem is now it doesnt want to farm at all it just wants to keep trying to continue on to the new quest  even if it chickens from that quest the first time. idk im tired ill mess with it more tomorrow just leave it off tonight then so chars dont rip lmao. goodnight bro thank you so much for all this work your putting into this and for free, your awesome bro.

**Reply by isid0re** [reply]
Oh that's a good point regarding farming vs questing for hardcore vs softcore. 

With softcore it's easier to be a glass cannon and push thru. I will definately look at the farmcheck module and adjust it regarding if gameplay is hardcore or not. Thanks.

As for the swapping and dropping. I will look closer at the smoke vs. Hp armor. I wanted life for survivability but also dont want to sacrifice my resistances. It is possible that I've weighted hp more than necessary. I'll defiantely check it out. Thsnks again.

**Reply by sifter456** [reply]
Yea like if it trys to do the next initial quest and chickens it should continue the next game mfing kind of like it was before how it woul do pits masouleum etc i know forsoftcore that could slow it down tremendously but for HC its actually a heavent send because they do get exp from the mf runs and better gear and have a better chance to succeed on the next quest.
